src/main/java/mate/academy/rickandmorty/model/Character.java

```diff
@@ -0,0 +1,26 @@
+package mate.academy.rickandmorty.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Data;
+
+@Data
+@Entity
```

Comment:
dont use data with entity as it cause bugs
```suggestion
@Getter
@Setter
@Entity
```